rust/datafusion/src/physical_plan/filter.rs

```diff
@@ -24,7 +24,7 @@ use std::sync::{Arc, Mutex};
 use crate::error::{ExecutionError, Result};
 use crate::physical_plan::{ExecutionPlan, Partitioning, PhysicalExpr};
 use arrow::array::BooleanArray;
-use arrow::compute::filter;
+use arrow::compute::filter_record_batch;
```

Comment:
This also improves performance because it avoids computing a `FilterContext` per column.

Reply:
@jorgecarleitao that's exactly right, good spot